logstash-input-neo4j.gemspec

```diff
@@ -0,0 +1,34 @@
+Gem::Specification.new do |s|
+
+  s.name            = 'logstash-input-neo4j'
+  s.version         = '0.9.0'
```

Comment:
because is the version that is already released to rubygems, https://rubygems.org/gems/logstash-input-neo4j. I start that way as I did a few iterations before this got finally released.